crates/aranya-client-capi/src/api/defs.rs

```diff
@@ -1356,3 +1296,266 @@ pub fn query_labels(
     *labels_len = data.len();
     Ok(())
 }
+
+/// A type containing the response of receiving a channel.
+#[aranya_capi_core::derive(Cleanup)]
+#[aranya_capi_core::opaque(size = 192, align = 8)]
+pub type AqcChannel = Safe<imp::AqcChannelType>;
+
+/// An enum containing all [`AqcChannel`] variants.
+#[repr(u8)]
+#[derive(Copy, Clone, Debug)]
+pub enum AqcChannelType {
+    Sender,
+    Receiver,
+    Bidirectional,
+}
+
+/// Bidirectional AQC Channel Object
+#[aranya_capi_core::derive(Cleanup)]
+#[aranya_capi_core::opaque(size = 184, align = 8)]
+pub type AqcBidiChannel = Safe<imp::AqcBidiChannel>;
+
+/// Sender AQC Channel Object
+#[aranya_capi_core::derive(Cleanup)]
+#[aranya_capi_core::opaque(size = 112, align = 8)]
+pub type AqcSenderChannel = Safe<imp::AqcSenderChannel>;
+
+/// Receiver AQC Channel Object
+#[aranya_capi_core::derive(Cleanup)]
+#[aranya_capi_core::opaque(size = 88, align = 8)]
+pub type AqcReceiverChannel = Safe<imp::AqcReceiverChannel>;
+
+/// Create a bidirectional AQC channel between this device and another peer.
+///
+/// Permission to perform this operation is checked against the Aranya policy.
+///
+/// @param client the Aranya Client [`Client`].
+/// @param team the team's ID [`TeamId`].
+/// @param peer the peer's network identifier [`NetIdentifier`].
+/// @param label_id the AQC channel label ID [`LabelId`] to create the channel with.
+/// @param channel the AQC channel object [`AqcBidiChannel`] that holds channel info.
+///
+/// @relates AranyaClient.
+pub unsafe fn aqc_create_bidi_channel(
+    client: &mut Client,
+    team: &TeamId,
+    peer: NetIdentifier,
+    label_id: &LabelId,
+    channel: &mut MaybeUninit<AqcBidiChannel>,
+) -> Result<(), imp::Error> {
+    // SAFETY: Caller must ensure `peer` is a valid C String.
+    let peer = unsafe { peer.as_underlying() }?;
+
+    let client = client.deref_mut();
+    let (chan, _) = client.rt.block_on(client.inner.aqc().create_bidi_channel(
+        team.into(),
+        peer,
+        label_id.into(),
+    ))?;
+
+    Safe::init(channel, imp::AqcBidiChannel::new(chan));
+
+    Ok(())
+}
+
+/// Delete a bidirectional AQC channel.
+///
+/// @param client the Aranya Client [`Client`].
+/// @param channel_id the AQC Channel [`AqcBidiChannel`] to delete.
+///
+/// @relates AranyaClient.
+pub fn aqc_delete_bidi_channel(
+    client: &mut Client,
+    channel: &AqcBidiChannelId,
```

Comment:
This arg doesn't match the docs. `channel_id` is more descriptive than `channel` so I'd pick that one.
```suggestion
    channel_id: &AqcBidiChannelId,
```

Reply:
This needs to be changed in ben's PR to be a channel since we aren't using IDs, I'm leaving it like this so I can get it to compile while I wait